build.sh

```diff
@@ -2,7 +2,7 @@
 source set_env.sh
 git submodule update --init --recursive
 cd firmware
-bash mpy_cross.sh
+bash mpy_cross.sh || exit 1
```

Comment:
Why? This does not match the description of the PR.

Reply:
Because I need to see where CI is failing.

Reply:
(And I'll remove it before a merge, of course)